### PR TITLE
feat(primitive): implement standard box

### PR DESCRIPTION
**Pull request by tarnas14**
Description
---
- [x] a box
- [x] uses theme border radius
- [x] interface for gradients
- [x] allows style customization
- [x] left usage examples in BaseNodeContainer (to be removed when implementing #7)

Motivation and Context
---

#83 

How Has This Been Tested?
---

image

### CODE REVIEW

File: applications/launchpad_v2/src/components/Box/index.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { StyledBox } from './styles'
+
+import { BoxProps } from './types'
+
+const Box = ({ children, gradient, border, style: inlineStyle }: BoxProps) => {
```
Comment (tomaszantas) [inline applications/launchpad_v2/src/components/Box/index.tsx:18]: NIT: JSDoc could be added here, but also the component name is self-explanatory, so can be skipped.

File: applications/launchpad_v2/src/components/Box/index.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  const style = {
+    border: border === false ? 'none' : undefined,
+    background:
+      gradient &&
+      `
+      linear-gradient(
+      45deg,
```
Comment (tomaszantas) [inline applications/launchpad_v2/src/components/Box/index.tsx]: NIT: Maybe the `45deg` could be also parametrised with props?

**Review by tomaszantas — DISMISSED** [review]
Looks good! two small NITs

File: applications/launchpad_v2/src/components/Box/index.tsx (hunk 1 above)
Comment (tarnas14) [reply applications/launchpad_v2/src/components/Box/index.tsx:18]: added

File: applications/launchpad_v2/src/components/Box/index.tsx (hunk 2 above)
Comment (tarnas14) [reply applications/launchpad_v2/src/components/Box/index.tsx]: added optional `rotation` to gradient, defaulting to 45

**Review by tomaszantas — APPROVED** [review]
Rubber stamp!

**Pull request merged**